src/compose/service.ts

```diff
@@ -1086,6 +1086,8 @@ export class Service {
 					log.warn(`Ignoring invalid bind mount ${volume}`);
 				}
 			} else {
+				// Push anonymous volume. Anonymous volumes can only be
+				// set by using the `VOLUME` instruction inside a dockerfile
```

Comment:
This is different than unmanaged volumes?

Reply:
what are unmanaged volumes? Anonymous volumes are a docker thing https://docs.docker.com/storage/volumes/#remove-anonymous-volumes, I think they are the reason we see sometimes those weirdly named volumes